.evergreen.yml

```diff
@@ -167,54 +193,41 @@ functions:
       script: |
         cp env/.cmdrc_sample.json env/.cmdrc.json
 
-  attach-results:
+  attach-test-results:
     command: attach.xunit_results
     params:
       files:
-        - "./spruce/bin/cypress/*.xml"
         - "./spruce/bin/jest/*.xml"
 
-  attach-cypress-screenshots:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/cypress/screenshots/*"]
-      remote_file: spruce/${task_id}/
-      bucket: mciuploads
-      content_type: image/png
-      permissions: public-read
-      display_name: "screenshot:"
-
-  attach-cypress-videos:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/cypress/videos/*"]
-      remote_file: spruce/${task_id}/
-      bucket: mciuploads
-      content_type: video/mp4
-      permissions: public-read
-      display_name: "video:"
-
-  attach-server-logs:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["evergreen/server_logs.txt"]
```

Comment:
I think there may have been a point in time when the evergreen log output was piped to a file. Logkeeper does this rn.